src/main/java/org/zalando/nakadi/Application.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"org.zalando.nakadi", "org.zalando.nakadi.config"})
```

Comment:
@adyach After upgrading spring boot, it would not find the org.zalando.nakadi.config.SecuritySettings component to create a bean, during tests. Authorization tests failed as a result.